homeassistant/components/bmw_connected_drive/device_tracker.py

```diff
@@ -16,12 +16,10 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     entities = []
 
     for vehicle in account.account.vehicles:
-        # if vehicle.state.is_vehicle_tracking_enabled:
         entities.append(BMWDeviceTracker(account, vehicle))
-        # else:
-        #     _LOGGER.info(
-        #         "Tracking is disabled for vehicle %s (%s)", vehicle.name, vehicle.vin
-        #     )
+        _LOGGER.info(
```

Comment:
This is now always shown, also when vehicle_tracking_enabled is true.

Reply:
Thanks, reintroduced the check and adjusted the error message (a device tracker can become available even if originally unavailable).